src/style.css

```diff
@@ -0,0 +1,43 @@
+body {
+  margin: 0;
+  min-height: 100vh;
+}
+
+.stars {
+  margin: auto;
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  margin-right: 4px;
```

Comment:
delete margin for the last star

Reply:
Thank you to check my mistakes, I fixed them a day ago but forgot to deploy link.